hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/HiveSyncConfigHolder.java

```diff
@@ -85,6 +85,10 @@ public class HiveSyncConfigHolder {
       .defaultValue("false")
       .markAdvanced()
       .withDocumentation("Skip the _ro suffix for Read optimized table, when registering");
+  public static final ConfigProperty<String> HIVE_SYNC_META_TO_ORIGIN_TABLE = ConfigProperty
+          .key("hoodie.datasource.hive_sync.sync_origin_table")
```

Comment:
Could you rename this config to `META_SYNC_SNAPSHOT_WITH_TABLE_NAME = … “hoodie.meta.sync.sync_snapshot_with_table_name”` and move it to `HoodieSyncConfig`, given the config can also apply to Glue sync which extends `HiveSyncTool`?